### PR TITLE
task/RTMD-33-session-key

### PR DESCRIPTION
**Pull request by bendiggle**
Created session key in config with default value set to hmrtm.sid
Pass config key in app.js

### CODE REVIEW

**Comment by bendiggle** [comment]
@JoeChapman @joefitter @gxxm Could you have a look please?

File: app.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ function initSession(req, res, next) {
     cookie: {
       secure: (req.protocol === 'https')
     },
-    key: 'hmbrp.sid',
+    key: config.session.key,
```
Comment (JoeChapman) [inline app.js]: Looks as if it might now be called `name` https://github.com/expressjs/session/blob/master/index.js#L89

File: config.js [hunk 2]
```diff
@@ -13,7 +13,8 @@ module.exports = {
   secureProtocol: process.env.SECURE_PROTOCOL || false,
   session: {
     secret: process.env.SESSION_SECRET || 'howdoesyourgardengrow',
-    ttl: process.env.SESSION_TTL || 1800 /* 30 mins timeout */
+    ttl: process.env.SESSION_TTL || 1800, /* 30 mins timeout */
+    key: process.env.SESSION_KEY || 'hmrtm.sid'
```
Comment (JoeChapman) [inline config.js]: See above `name`

**Comment by bendiggle** [comment]
@JoeChapman I have updated this. Is this ok now?

**Comment by bendiggle** [comment]
@JoeChapman @joefitter @gxxm 👀

**Comment by JoeChapman** [comment]
looking

File: config.js [hunk 3]
```diff
@@ -14,7 +14,8 @@ module.exports = {
   secureProtocol: process.env.SECURE_PROTOCOL || false,
   session: {
     secret: process.env.SESSION_SECRET || 'howdoesyourgardengrow',
-    ttl: process.env.SESSION_TTL || 1800 /* 30 mins timeout */
+    ttl: process.env.SESSION_TTL || 1800, /* 30 mins timeout */
+    name: process.env.SESSION_KEY || 'hmrtm.sid'
```
Comment (JoeChapman) [inline config.js]: What does `hmrtm` stand for? Could `rotm.sid` be more relevant, or perhaps `hodrotm.sid`?

Comment (bendiggle) [inline config.js]: `rotm.sid` would be better. I was just following the same conventions as gro. Will update now

**Comment by bendiggle** [comment]
@JoeChapman Updated

**Comment by joefitter** [comment]
👍 from me

**Comment by JoeChapman** [comment]
cool

**Pull request merged**